portfolio/src/main/webapp/script.js

```diff
@@ -116,13 +116,14 @@ function createListElement(text) {
 }
 
 /** 
- * Returns HTML image element from Url 
+ * Returns HTML image element from url.
  */
-function createImageElement(url) {
-  if (!url) 
+function createImageElement(imageString) {
+  if (!imageString) 
     return null;
   const imgElement = document.createElement('img');
-  imgElement.src = Url;
+  // send GET request to DisplayBlobServlet to serve the image with imageString as blobKey string
```

Comment:
I know this is just a comment and is a bit nit-picking, but actually this is just creating the HTML right?  The browser might never make the GET request to the servlet.

Reply:
That's right, I changed the comment.